README.md

```diff
@@ -0,0 +1,73 @@
+### FAQ
+
+змінити файл web/src/App.js , web/src/index.css
```

Comment:
@ArtemBernatskyy , your Readme.md file doesn't look fine. It is written in Ukrainian language, but English language is preferable. Also it does not contain instructions about how to setup and run the project.